.github/operator-release-files/operatorhub-bundle/hazelcast-enterprise-operator.vOPERATOR_VERSION.clusterserviceversion.yaml

```diff
@@ -44,13 +44,11 @@ spec:
 
     ### Before Your Start
 
-    Add a Secret within the Project that contains the Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
-
-        $ oc create secret generic hz-license-key-secret --from-literal=key=LICENSE-KEY-HERE
```

Comment:
Isn't that line needed any more?

Reply:
Thanks for the review 👍 

I have already added same informations/directions into RHM/OperatorHub instructions based on RedHat guys' advise so I removed that part from dashboard.